tutorials/hello-kubernetes/README.md

```diff
@@ -92,7 +203,9 @@ kubectl apply -f ./deploy/redis.yaml
 component.dapr.io/statestore created
 ```
 
-## Step 3 - Deploy the Node.js app with the Dapr sidecar
+> Note: If you installed Dapr in the Dev mode in Kubernetes, then the statestore component will be created automatically in the `default` namespace. The above commmand will output `component.dapr.io/statestore configured` instead of `component.dapr.io/statestore created`.
```

Comment:
```suggestion
> Note: If you installed Dapr in `--dev` mode in Kubernetes, then the statestore component is created in the `default` namespace. The above command outputs`component.dapr.io/statestore configured` instead of `component.dapr.io/statestore created`.
```